packages/fluentui/react-northstar/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -155,19 +155,21 @@ const DropdownSelectedItem: React.FC<WithAsProp<DropdownSelectedItemProps>> &
           }),
       };
 
-  const imageProps = _.isNil(image)
-    ? image
-    : {
-        children: (ComponentType, props) =>
-          Image.create(image, {
-            defaultProps: () => ({
-              avatar: true,
-              className: dropdownSelectedItemSlotClassNames.image,
-              styles: resolvedStyles.image,
+  const imageChildren = _.get(image, 'children');
```

Comment:
I tried in previously commit to solve it using `childrenExist` but it will be true for text only and then not working as expected. If you don't agree with this solution, have any other idea? @layershifter and @pompomon